src/app/visual/date-pipe/date-pipe-visual.component.ts

```diff
@@ -46,15 +55,30 @@ export class DatePipeVisualComponent implements OnInit {
 
   public myDate = new Date('11/05/1955');
 
+  private ngUnsubscribe = new Subject<void>();
+
   constructor(
-    private datePipe: SkyDatePipe
-  ) { }
+    private datePipe: SkyDatePipe,
+    private localeProvider: SkyAppLocaleProvider
+  ) {
+    // Update locale to the browser's current locale.
+    this.localeProvider.getLocaleInfo()
```

Comment:
Isn't this done automatically by the pipe (and Builder)?
See: https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/date-pipe/date.pipe.ts#L44-L49

I thought the public `locale` variable in the visual demo was being used as an override to the browser's current culture?

Reply:
> I thought the public locale variable in the visual demo was being used as an override to the browser's current culture?

Correct. The only thing the demos weren't doing was showing you (in the template) what locale you were currently using on load. For example, the dropdown menu should now show your current locale and the displayed formatting will respect the browser's current locale on load. Before this change, it was always initializing with `en-US`.

Reply:
Even if the `locale` was left `undefined`?

Reply:
If the `locale` is `undefined` the pipe works correctly (shows the browser's current locale`, but the visual test doesn't show which local is selected in the dropdown. That's really the only reason I did this - it was just a nicety.

Maybe overkill?